packages/concerto-tools/types/lib/codegen/fromcto/protobuf/protobufvisitor.d.ts

```diff
@@ -43,21 +43,26 @@ declare class ProtobufVisitor {
      */
     public createImportLineStrings(imports: any[]): string[];
     /**
-     * Get the names of the children of a class.
-     * @param {string} className - the name of the class
-     * @param {Object[]} declarations - the declarations in scope
-     * @return {string[]} an array of the names of the children of the classes
+     * Recursively get the names of the subclasses of a class.
+     * @param {Object} classDeclaration - the class declaration object
+     * @return {String[]} an array of the names of the subclasses of the class
      * @public
      */
-    public getChildrenOfClass(className: string, declarations: any[]): string[];
+    public getNamesOfSubclassesOfClassRecursively(classDeclaration: any): string[];
```

Comment:
What's the motivation for this rename? The function name is getting very long!

Reply:
Hmm, how did this end up in here? 🤔 Probably some code left over on the chopping board from my Protobuf work. Reverted ✅